pkgs/games/colobot/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, lib, fetchFromGitHub, cmake, boost, SDL2, SDL2_image, SDL2_ttf, libpng
+, glew, gettext, libsndfile, libvorbis, libogg, physfs, openal
+, xmlstarlet, doxygen, python3, colobot-data }:
+
+stdenv.mkDerivation rec {
+  pname = "colobot";
+  version = "0.1.12-alpha";
```

Comment:
We should probably add a comment to also update the data derivation and vice versa.

Reply:
@freezeboy Ideally the comment saying updates to this derivation should go along with updates to `./data.nix`, should be here, right above the version, and it shouldn't have typos :)